haskell/src/Ligo/BaseDAO/Types.hs

```diff
@@ -706,8 +697,8 @@ mkFullStorage
   :: "admin" :! Address
   -> "votingPeriod" :? VotingPeriod
   -> "quorumThreshold" :? QuorumThreshold
-  -> "maxChangePercent" :? MaxChangePercent
-  -> "changePercent" :? ChangePercent
+  -> "maxQuorumFraction" :? QuorumFraction
```

Comment:
"maxQuorumFraction" does not sound much like it is representing "maxChangePercent". Any particular reason why this was changed?

Reply:
No good reason, this was changed by accident :sweat_smile: 

Fixed now